src/main/java/ExpiredTokenJob.java

```diff
@@ -0,0 +1,53 @@
+import fr.maif.socleapi.croupier.model.TokenInfos;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class ExpiredTokenJob {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExpiredTokenJob.class);
+
+    @Autowired
+    private TokenService tokenService;
+
+    @Autowired
+    private MailService mailService;
+
+    final String helloBlabla;
+
+    private static final long MONTH_MS = 2628000000L;
+
+    @Scheduled(fixedRate = 604800000)
+    public void checkExpirationDate() {
+        List<TokenInfos> expiredTokens = tokenService
+                .getTokens()
+                .stream()
+                .filter(token -> token.getExpirationDate().getTime() < (new Date(System.currentTimeMillis()).getTime() + MONTH_MS))
```

Comment:
Consider extracting the logic inside the lambda expression into a separate method for better readability and maintainability.